### DISCUSSION TITLE
Google Summer Of Code 2024

### DISCUSSION THREAD

**Discussion by shivarm** [Hyperfoil/Horreum#1088] [General]
@johnaohara @jesperpedersen will horreum taking part in Google Summer Of Code 2024?

**Comment by johnaohara** [comment]
@shivam-sharma7 Unfortunately at this point in time, I do not think we will have enough capacity to support a GSoC entry for Horreum for 2024

**Reply by shivarm** [reply]
@johnaohara Thanks, I need to look at another project for career.

**Comment by stalep** [comment]
I have to agree with John here, atm we do not have the capacity to support GSoC.

**Reply by shivarm** [reply]
Thanks @stalep

**Comment by shivarm** [comment]
> capacity

?

**Reply by johnaohara** [reply]
In order to participate, we need to make a submission, mentor etc. that all takes time. Our team is currently staffed at 70% of what is was, with the same deliverables for Red Hat.  At this point in time, unfortunately we do not have enough hours to be able to participate

@jesperpedersen is currently working on other projects (hi jesper ;) )

**Comment by shivarm** [comment]
@johnaohara @jesperpedersen @stalep But this communit helped me lot. Would you say i have done some works here I? 
Thanks

**Reply by johnaohara** [reply]
@shivam-sharma7 your contributions have been really appreciated. It is just unfortunate that we do not currently have the number of pple on the project to be able to participate in GSoC for 2024

**Reply by shivarm** [reply]
Yes!
